### DISCUSSION TITLE
Asian Gong scenery object plays "Gong" sound effect when clicked

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#18540] [Feature requests]
I'd like if the Asian Gong scenery object when clicked would play a gong sound effect. Similar to how clicking on baloons pops them or ducks makes them quack

**Comment by ocalhoun6** [comment]
Theoretically, this could be done for a lot of different scenery objects.

It would be a lot of work (and extra data in the OpenRCT2 filesize for the sound files) for questionable/marginal benefit. Pretty rare for players to go around clicking on scenery objects, right?

That said, it would be cool if custom scenery objects could have an `OnClick` action chosen by the designer of the object. It could play a sound (either a default one or one provided with the object), it could trigger certain animation frames in the object, or maybe it could trigger a plugin hook ... which would then allow people to make clicking on the scenery do damn near anything by combining it with a plugin.

(Actually, come to think of it ... this 'play a sound when certain scenery items are clicked on' sounds like the kind of thing a plugin might be able to do well... Just need a plugin hook for scenery being clicked on + make sure plugins are able to play sound files.)

**Reply by karst** [reply]
The sound effects are all really small since they're so short, only a few kb each. I'd definitely keep it rare though
